.travis.yml

```diff
@@ -8,12 +8,15 @@ sudo: false
 # http://docs.travis-ci.com/user/caching/
 cache: pip
 
-
 env:
-  - SPHINXOPTS="-W"
+  - SPHINXOPTS="-W" SPHINX_VERSION=1.2.3
+  - SPHINXOPTS="-W" SPHINX_VERSION=latest
 
 before_install:
-  - pip install Sphinx==1.2.3
+  - if [[ $SPHINX_VERSION == '1.2.3' ]]; then pip install Sphinx==1.2.3; fi
```

Comment:
Why not `if [[ $SPHINX_VERSION != 'latest' ]]; then pip install Sphinx==$SPHINX_VERSION; fi` for flexible future use?

Reply:
That's fine, happy to push the comment. Note the objective is to fully drop the `SPHINX_VERSION` variable when the OMERO documentation is updated to build with the latest Sphinx release as are currently doing across the board.